### ISSUE TITLE
add edition capabilities to state inspector

### ISSUE THREAD

**Issue by avilleret** [ossia/score#921]
it would be awesome to be able to make copy/paste in the state inspector and to edit addresses

**Comment by bltzr** [comment]
yeah ! and paste/replace (e.g. to replace /address.1 with /address.2)
